verification/testNF90io/input/gendata.py

```diff
@@ -0,0 +1,94 @@
+from numpy import *
+from scipy import *
+from pylab import *
+from shutil import copy
+import numpy.matlib as matlib
```

Comment:
Check.  Cleaned this script up a bit. See new commit.  

FWIW, I think the capabilities of `nf90io` should just be tested inside one of the verification examples that does something useful rather than this example. But I'll leave organizing such tests to the maintainers who have a much better overview of the issues than I do.